python/pyarrow/__init__.py

```diff
@@ -149,6 +148,11 @@ def parse_git(root, **kwargs):
                          serialize_pandas, deserialize_pandas)
 import pyarrow.ipc as ipc
 
+try:
+    from pyarrow.plasma import PlasmaObjectExists, PlasmaStoreFull
```

Comment:
We shouldn't import plasma from here. People will have to import those exceptions from `arrow.plasma` instead.

Reply:
OK, I was trying to maintain a little backwards compatibility but i'll move them.

Reply:
removed.